composer.json

```
@@ -17,7 +17,7 @@
   },
   "require-dev": {
     "ergebnis/composer-normalize": "^2.42",
-    "friendsofphp/php-cs-fixer": "^3.11",
+    "friendsofphp/php-cs-fixer": "^3.6",
```

The changes were introduced in `3.60`, not `3.6` ;)
```suggestion
    "friendsofphp/php-cs-fixer": "^3.60",
```